app/presenters/image_upload_response_presenter.rb

```diff
@@ -11,7 +11,11 @@ def success?
   end
 
   def errors
-    @form_response.errors.except(:hints).flat_map do |key, errs|
+    form_response_errors = @form_response.errors.clone
+    if form_response_errors.many? { |k, v| %i[name dob dob_min_age state].include?(k) }
+      form_response_errors = { pii: [I18n.t('doc_auth.errors.general.no_liveness')] }
```

Comment:
Nice use of `.many?`

Reply:
Another option is to deal this in the doc_pii_form, so it's the form's responsibility to deal with all this?

Reply:
@dawei-nava  - yes, this consolidation seems to be more about what the FE receives.  i did look at perforning this in the DocPIIForm but I figured this way allows us to be more explicit in our analytics and support